app/src/ui/lib/context-menu.ts

```diff
@@ -13,6 +13,10 @@ export const CopySelectedRelativePathsLabel = __DARWIN__
   ? 'Copy Relative Paths'
   : 'Copy relative paths'
 
+export const CopyRepositoryLocationLabel = __DARWIN__
```

Comment:
👋 @novialriptide first-time contributor 🎉 

It works as expected. The only thing I can suggest is to remove `CopyRepositoryLocationLabel` because it's not in use anywhere.

Reply:
Done!